c/parallel/include/cccl/c/types.h

```diff
@@ -92,5 +80,4 @@ struct cccl_iterator_t
   cccl_op_t dereference;
   cccl_type_info value_type;
   void* state;
-  cccl_string_views* ltoirs = nullptr;
```

Comment:
Because we compile the `op` for `TransformIterator` on the Python side, we don't need to pass the LTOIR here.

Generally speaking, any device functions used by the `advance` and `dereference` methods can be compiled purely on the Python side.